### PR TITLE
update to latest ubi version

### PR DESCRIPTION
**Pull request by amiralig**
As per https://docs.sonatype.com/display/ENG/Lifecycle+Policy+Violation+Handling+Process, when Neuvector finds violations, we update to the latest base image before releasing.

### CODE REVIEW

**Review by guillermo-varela — APPROVED** [review]
+1

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-210
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-212
```
Comment (mealingr) [inline Dockerfile]: A new 8.5-204 version got released ~10 hours ago, we could try that too
https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?tag=8.5-204&push_date=1636504224000

**Review by jflinchbaugh — APPROVED** [review]
+1

**Review by fernau — APPROVED** [review]
LGTM - maybe we could go straight to 8.5-204 ?

**Review by koraytugay — COMMENTED** [review]
I agree with @fernau and @mealingr

**Review by koraytugay — APPROVED** [review]
+1

File: Dockerfile (hunk 1 above)
Comment (amiralig) [reply Dockerfile]: resolved sorry for the delay

**Pull request merged**